src/sentry/api/serializers/models/group.py

```diff
@@ -347,8 +349,20 @@ def serialize(self, obj, attrs, user):
         # do not return the permalink which contains private information i.e. org name.
         request = env.request
         is_superuser = request and is_active_superuser(request) and request.user == user
-        if is_superuser or (
-            user.is_authenticated() and user.get_orgs().filter(id=obj.organization.id).exists()
+
+        # If user is a sentry_app then it's a proxy user meaning we can't do a org lookup via `get_orgs()`
+        # because the user isn't an org member. Instead we can use the auth token and the installation
+        # it's associated with to find out what organization the token has access to.
+        is_valid_sentryapp = False
+        if hasattr(request.user, "is_sentry_app") and isinstance(request.auth, ApiToken):
```

Comment:
@MeredithAnya We should probably use `getattr` here since this would return true if `user.is_sentry_app == False`

Reply:
@scefali yup, will update